_posts/2018-03-22-industrialization-versus-data-selection.md

```diff
@@ -0,0 +1,47 @@
+
+## ASR: Automatic Speech Recognition Prelude
+
+In ASR, we always consider building two independent modules which will be mixed together later on. Each module in itself is very dependent of the language we want to recognize.
+
+- The first one is called acoustic model. It represents the way of speaking. What sounds can be put together to form a word, a sentence. In fact, we represent the language by a series of phoneme. If we watch in [Wiktionary](https://en.wiktionary.org/wiki/phoneme) it's clearer isn't it? Don't confuse with syllable through. We can take an example. The word 'through'(1 syllable) consists of three sounds, three phonemes: 'th' 'r' 'oo'. So the aim is to models the sounds as a sequence of phonemes.
+- The second one is the language model, which we want to build automatically. It models the distribution of the words for a given language. Thanks to those probabilities, the LM helps picking the best correspondence between a sequence of phonemes and the words/sentences.
+
+In order to build these modules, we need data and the more we have and the more they are relevant, the better! That's why we need data selection: we need a mean to retrieve relevant data adapted to the context of the recognition. In fact a lawyer and a baker don't speak the same language: they don't use the same lexicon. The data selection is picking the good data that match the domain within millions of examples through the usage of various automatic algorithms.
+
+## How we used to do
+
+As we discussed above, data selection is a very important step while building a system. Like many, we used the [Moore-Lewis method](http://www.aclweb.org/anthology/P10-2041) which was also adapted for bilingual use (like translation) by Axelrod et al. in [Domain Adaptation via Pseudo In-Domain Data Selection](https://aclanthology.info/pdf/D/D11/D11-1033.pdf). These are very effective ways to select data using two corpora (in and out domain) by comparing cross-entropies. In domain meaning that the corpus have specific data, that are relevant with the context, the domain of the recognition as explained before with the lawyer/baker thing. Whereas out domain is just a pool of random data, meaning there is relevant and no-relevant data in it! Then about the cross-entropy, it's the method that help choosing the well-matched data for the desired output. Thanks to some relevant segment, we compare each segment in the pool of data to retrieve the closest to the initial data.
+
+This method is not really scalable because the algorithm can't decide when to stop on its own and he has an annoying tendency to promote very short to short sentences. Moreover, something hit us hard. This article turned out to be eight this years and we have never searched for an other method before. So we asked ourselves: has any new work been done in data selection since this article? And are there any relevant work ready for a more industrialized turn?
```

Comment:
"This method" = cross-entropy one? Or the whole in+out+cross-entropy?

Reply:
"this article" = Axelrod's one?

Reply:
"eight this years" => "eight this year"

Reply:
"we have never searched for" => looked for?

Reply:
"an other" => "another"

Reply:
General comment: "article" is not really the right word, you should use "paper", or at least mix the two

Reply:
"And are there" => "And is there"